recipe/meta.yaml

```diff
@@ -1,34 +1,38 @@
 {% set name = "pychrono" %}
 {% set version = "4.0.0" %}
-{% set build = 2 %}
-{% set sha256 = "803b5089e48334771f652f5bec79f95f2a42cfb02d2ffbaf8ad9e5dca2ba1886" %}
+{% set build = 3 %}
+{% set sha256 = "ccaaabc4359a245ecef211f1d7405f8d8554d5a9dd7145d0e283e31736b56b5f" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/projectchrono/chrono/archive/{{ version }}.tar.gz
+  url: https://github.com/projectchrono/chrono/tarball/develop
+  fn: develop.tar.gz
   sha256: {{ sha256 }}
 
 build:
   number: {{ build }}
-  skip: true  # [not linux]
+  skip: true  # [win or py2k]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
-    - swig 3.0.12
+    - swig 4.0.1
   host:
     - python
+    - eigen
+    - openmp
```

Comment:
See https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp

Reply:
Thanks @isuruf, I will add that in the next commit.